.github/workflows/build_windows_exe.yml

```diff
@@ -0,0 +1,20 @@
+name: Build_Windows_exe
+
+on:
+  push:
+    tags:
+    - "*"
+
+jobs:
+  Build_Windows_exe:
```

Comment:
Mind to make this lower case? Also on line 1, by the way.

Reply:
I did this to match Checks and Test GitHub Actions that are both in Title case.

Reply:
Okay, I see. – Only the `name` attribute is not lower case, though (see [Checks](https://github.com/PythonTurtle/PythonTurtle/blob/master/.github/workflows/checks.yml#L1), [Tests](https://github.com/PythonTurtle/PythonTurtle/blob/master/.github/workflows/tests.yml#L1), [Publish](https://github.com/PythonTurtle/PythonTurtle/blob/master/.github/workflows/packaging.yml#L1)).